twinkles/twinklesCatalogDefs.py

```diff
@@ -48,3 +48,37 @@ def get_phoSimMagNorm(self):
                 magNorm_out = magNorm + self.column_by_name(varName)
 
             return magNorm_out
+
+    def _query_and_write(self, filename, chunk_size=None, write_header=True,
```

Comment:
Oh, I will remove that. That was from an earlier attempt to get this to work. Thanks.